### PR TITLE
Normalisation des coicop sur categories fiscales calées

### PR DESCRIPTION
**Pull request by MalkIPP**

### CODE REVIEW

File: openfisca_france_data/input_data_builders/build_openfisca_indirect_taxation_survey_data/step_04_homogeneisation_categories_fiscales.py [hunk 1]
```diff
@@ -66,7 +76,12 @@ def build_menage_consumption_by_categorie_fiscale(year_data = None, year_calage
         categorie_fiscale_by_coicop.get(coicop)
         for coicop in coicop_data_frame.columns
         ]
+<<<<<<< HEAD
     # print categorie_fiscale_labels
+=======
+    #TODO: gérer les catégorie fiscales "None" = dépenses énergétiques (4) & tabac (2)
+#    print categorie_fiscale_labels
+>>>>>>> Apply normalization to categories fiscales
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_indirect_taxation_survey_data/step_04_homogeneisation_categories_fiscales.py]: @MalkIPP  Unresolved conflict

File: openfisca_france_data/input_data_builders/build_openfisca_indirect_taxation_survey_data/step_03_calage.py [hunk 2]
```diff
@@ -44,23 +46,34 @@
...
+#    11 Hotels, cafés, restaurants
+#    12 Biens et services divers
+        if grosposte != 99:
+            ratio_bdf_cn = masses.at[
+                                    grosposte,'ratio_bdf{}_cn{}'.format(year_data, year_data)
+                                    ]
+            ratio_cn_cn = masses.at[
+                                    grosposte,'ratio_cn{}_cn{}'.format(year_data, year_calage)
+                                    ]
+            depenses_calees[column] = depenses[column]*ratio_bdf_cn*ratio_cn_cn
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_indirect_taxation_survey_data/step_03_calage.py]: Indentation ...

File: openfisca_france_data/input_data_builders/build_openfisca_indirect_taxation_survey_data/step_0_1_1_homogeneisation_donnees_depenses.py [hunk 3]
```diff
@@ -266,14 +265,27 @@ def normalize_coicop(code):
...
     elif len(code) == 4:
-        if not code.startswith("1") and not code.startswith("9"):
+        if not code.startswith("0") and not code.startswith("1") and not code.startswith("45") and not code.startswith("9"):
             normalized_code = "0" + code
-        elif code in ["1151", "1181"]:
+            # 022.. = cigarettes et tabacs => on les range avec l'alcool (021.0)
+        elif code.startswith("0"):
+            normalized_code = code + "0"
+        elif code in ["1151", "1181","4522","4511"]:
+            # 1151 = Margarines et autres graisses végétales
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_indirect_taxation_survey_data/step_0_1_1_homogeneisation_donnees_depenses.py]: spaces after commas

**Comment by MalkIPP** [comment]
Corrected

**Pull request merged**